manifest.toml

```diff
@@ -1,9 +1,5 @@
 name = "C++"
 language = "cpp"
+aliases = ["C++", "C"]
```

Comment:
I would suggest adding `cuda` here as well.

 .cu/.cuh are CUDA [C source files](https://github.com/Haybla/LDPC-CC_Pipeline_Decoder/blob/d814a76df98ebc66a6f94a8595d34e4ab04906ac/src/kernel.cu)/headers for NVCC compiler that will be detected as a separate `Cuda` language.

Reply:
Makes total sense, thanks!

Will also check some other C-like languages from the [list](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml).

Reply:
Done!